source/android/mobile/src/main/java/io/adaptivecards/adaptivecardssample/MainActivityAdaptiveCardsSample.java

```diff
@@ -371,10 +408,27 @@ private String loadFile(Uri uri)
             return null;
         }
 
-        InputStream inputStream = null;
         try
         {
-            inputStream = getContentResolver().openInputStream(uri);
+            return readStream(getContentResolver().openInputStream(uri));
+        }
+        catch (FileNotFoundException e)
```

Comment:
Doesn't this code trip static code analysis warnings as errors today around unused variables? e isnt being used as far as i can tell so just dont capture it? Does java syntax not allow for this (am comparing iwht c#)?

Reply:
Yeah, you need to capture the variable if you'd like to catch that type of error in Java, even if you don't intend to use e (btw, this is just existing code that I moved here)